resources/authgear/templates/en/web/authflowv2/__load_bot_protection.html

```diff
@@ -0,0 +1,7 @@
+{{ if $.BotProtectionRequired }}
+  {{ if (eq $.BotProtectionProviderType "cloudflare") }}
+    <script nonce="{{ $.CSPNonce }}" src="https://challenges.cloudflare.com/turnstile/v0/api.js?render=explicit"></script>
```

Comment:
Support the languages https://developers.cloudflare.com/turnstile/reference/supported-languages/